models/issue_reaction.go

```diff
@@ -33,16 +34,38 @@ type FindReactionsOptions struct {
 }
 
 func (opts *FindReactionsOptions) toConds() builder.Cond {
+	//If Issue ID is set add to Query
 	var cond = builder.NewCond()
 	if opts.IssueID > 0 {
 		cond = cond.And(builder.Eq{"reaction.issue_id": opts.IssueID})
 	}
+	//If CommentID is > 0 add to Query
+	//If it is 0 Query ignore CommentID to select
+	//If it is -1 it explicit search of Issue Reactions where CommentID = 0
 	if opts.CommentID > 0 {
 		cond = cond.And(builder.Eq{"reaction.comment_id": opts.CommentID})
+	} else if opts.CommentID == -1 {
+		cond = cond.And(builder.Eq{"reaction.comment_id": 0})
 	}
+
 	return cond
 }
 
+// FindCommentReactions returns a ReactionList of all reactions from an comment
+func FindCommentReactions(comment *Comment) (ReactionList, error) {
+	return findReactions(x, FindReactionsOptions{
```

Comment:
I really do think we should be censoring the reactions that are no longer allowed here.

Reply:
I think this looks like a "data loose" to users ...
... thats why I allow to remove all reaction types (you can remove not allowed but dont add new anymore)

Reply:
As I wrote on discord, there are emoji that can be used in with racist intent - a server administrator might discover that previously allowed reactions were being used in this way and as such although they might delete the user(s) responsible for safety might just want to disable all use of these.

Reply:
This can be thought about in another PR.

Reply:
👍 ->  #9286